ui/packages/catalog-ui-search/src/main/webapp/component/result-form/result-form.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+ /*global define,window*/
```

Comment:
✏️ Pretty sure these comments are only to tell jshint about globally defined variables so it doesn't freak out if they're used without declaring them, and we don't use jshint anymore